apps/hashdotdev/src/_pages/docs/apps/01_develop.mdx

```diff
@@ -0,0 +1,9 @@
+# Developing Apps
+
+**HASH Apps have not yet launched. Documentation regarding their creation will be added here prior to release.**
+
+If you would like to be notified when HASH Apps launch, fill in your email address below:
+
+---
+
+_By signing up you agree to our Privacy Policy, and will be invited to reserve your HASH app username._
```

Comment:
There's no signup form on these pages – I assume that's known. Once it exists we could add this line in the form rather than each of these pages.